### ISSUE TITLE
Update to Carpentries Workbench

### ISSUE THREAD

**Issue by colinsauze** [carpentries-incubator/machine-learning-novice-sklearn#43]
This lesson should be updated to the new Carpentries workbench format. There's a tool to automate this available from https://github.com/carpentries/lesson-transition#readme

**Comment by josenino95** [comment]
@colinsauze I'm preparing to teach this workshop in February, 2025 at the University of California Santa Barbara, and I actually was thinking of updating the workshop to the workbench for that. So if that's ok with you, I'd like to take over this issue and submit a PR in the coming weeks

**Comment by colinsauze** [comment]
Yes it would be great to get it converted to workbench format if you're able to do that. I know there's an automatic conversion tool, but I'm not sure if it works with incubator repositories. I will be interested to hear your experiences if you use this.

**Comment by colinsauze** [comment]
Now converted to workbench.
